enterprise/internal/batches/store/batch_changes_test.go

```diff
@@ -20,27 +20,30 @@ import (
 )
 
 func testStoreBatchChanges(t *testing.T, ctx context.Context, s *Store, clock ct.Clock) {
-	cs := make([]*btypes.BatchChange, 0, 3)
+	cs := make([]*btypes.BatchChange, 0, 4)
 
 	t.Run("Create", func(t *testing.T) {
 		for i := 0; i < cap(cs); i++ {
 			c := &btypes.BatchChange{
 				Name:        fmt.Sprintf("test-batch-change-%d", i),
 				Description: "All the Javascripts are belong to us",
 
-				InitialApplierID: int32(i) + 50,
-				LastAppliedAt:    clock.Now(),
-				LastApplierID:    int32(i) + 99,
-
 				BatchSpecID: 1742 + int64(i),
 				ClosedAt:    clock.Now(),
 			}
 
-			if i == 0 {
+			if i <= 1 {
 				// Check for nullability of fields by not setting them
 				c.ClosedAt = time.Time{}
```

Comment:
```suggestion
				c.LastAppliedAt = time.Time{}
```

Two bird- things? with one diff?

Anyway: since the first one is a draft we can make that explicit by setting `LastAppliedAt` to a zero-value *and* check nullability of fields (as commented here)